Warn serverless forwarder users (#13055)
DataDog/documentation: content/en/serverless/guide/datadog_forwarder_node.md

kind: guide
---

<div class="alert alert-warning">

> If you are a new user of Datadog Serverless, follow the <a href="/serverless/installation/nodejs">instructions to instrument your Lambda functions using the Datadog Lambda Extension</a> instead. However, if you got set up on Datadog Serverless using the Datadog Forwarder before Lambda offered out-of-the-box functionality, use this guide to maintain your instance.

Reviewer: ```suggestion
If you are a new user of Datadog Serverless, follow the <a href="/serverless/installation/nodejs">instructions to instrument your Lambda functions</a> using the Datadog Lambda Extension instead. If you have setup Datadog Serverless with the Datadog Forwarder before Lambda offered out-of-the-box functionality, use this guide to maintain your instance.
```
